Libraries/Amazon.Lambda.Annotations.slnf

```diff
@@ -2,11 +2,36 @@
   "solution": {
     "path": "Libraries.sln",
     "projects": [
-      "src\\Amazon.Lambda.APIGatewayEvents\\Amazon.Lambda.APIGatewayEvents.csproj",
       "src\\Amazon.Lambda.Annotations.SourceGenerator\\Amazon.Lambda.Annotations.SourceGenerator.csproj",
       "src\\Amazon.Lambda.Annotations\\Amazon.Lambda.Annotations.csproj",
+      "src\\Amazon.Lambda.APIGatewayEvents\\Amazon.Lambda.APIGatewayEvents.csproj",
```

Comment:
Why were all of these other packages added to the Annotations solution filter? I can't see why we would need all of these event projects or the ASP.NET Core bridge library.

Reply:
I don't know how that happened, removed.

Reply:
Doesn't look like you have pushed your change to remove all of these extra projects to the solution filter.

Reply:
I also think we need Amazon.Lambda.RuntimeSupport added to the solution filter now:


```
Severity	Code	Description	Project	File	Line	Suppression State
Error	NU1105	Unable to find project information for 'C:\dev\aws-lambda-dotnet\Libraries\src\Amazon.Lambda.RuntimeSupport\Amazon.Lambda.RuntimeSupport.csproj'. If you are using Visual Studio, this may be because the project is unloaded or not part of the current solution so run a restore from the command-line. Otherwise, the project file may be invalid or missing targets required for restore.	Amazon.Lambda.Annotations.SourceGenerators.Tests	C:\dev\aws-lambda-dotnet\Libraries\test\Amazon.Lambda.Annotations.SourceGenerators.Tests\Amazon.Lambda.Annotations.SourceGenerators.Tests.csproj	1	
Error	NU1105	Unable to find project information for 'C:\dev\aws-lambda-dotnet\Libraries\src\Amazon.Lambda.RuntimeSupport\Amazon.Lambda.RuntimeSupport.csproj'. If you are using Visual Studio, this may be because the project is unloaded or not part of the current solution so run a restore from the command-line. Otherwise, the project file may be invalid or missing targets required for restore.	TestExecutableServerlessApp	C:\dev\aws-lambda-dotnet\Libraries\test\TestExecutableServerlessApp\TestExecutableServerlessApp.csproj	1	
```

Reply:
Updated, thanks @ashovlin